po/extra/pt.po

```diff
@@ -0,0 +1,360 @@
+msgstr ""
+"Corrige o aumento do tamanho da janela quando o resumo ou nome do pacote é "
+"muito comprido"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:128
+msgid "The application can be now found under \"System\" category"
+msgstr "A aplicação pode agora ser encontrada na categoria \"Sistema\""
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:129
+msgid "Ctrl-Q now exists the app"
+msgstr "Ctrl-Q agora faz sair da aplicação"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:136
+msgid "Do not display error dialog when operation cancelled"
+msgstr "Não mostra a caixa de diálogo de erro quando a operação é cancelada"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:137
+msgid "Display the progress of current operation in the dock"
+msgstr "Mostrar o progresso da operação atual na Doca"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:138
+msgid "Sort packages by their state and name in the list view"
+msgstr "Ordena os pacotes por estado e nome na visualização de lista"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:145
+msgid "Save window state between sessions"
+msgstr "Guardar o estado da janela entre as sessões"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:146
+msgid "Improved performance of launching the app second time"
+msgstr "Melhor desempenho ao iniciar a aplicação pela segunda vez"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:153
+msgid "Support other package formats such as .rpm for other package managers"
+msgstr ""
+"Suporta outros formatos de pacote como .rpm para outros gestores de pacotes"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:154
+msgid "Minor fixes"
+msgstr "Pequenas correções"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:161
+msgid "Added an ability to open packages with Eddy in the file manager"
+msgstr ""
+"Adicionada a capacidade de abrir pacotes com o Eddy no gestor de ficheiros"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:162
+msgid "Show a spinner when loading packages"
+msgstr "Mostrar um botão giratório ao carregar pacotes"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:163
+msgid "Fixed adding a file already in the list duplicated it"
+msgstr "Corrige o adicionar de um ficheiro já na lista de o duplicar"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:170
+msgid "Completely port to PackageKit and remove aptdaemon dependency"
+msgstr ""
+"Mudança completa para o PackageKit e remove a dependência do aptdaemon"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:171
+msgid "Fix wrong icon displayed in the notifications"
+msgstr "Corrige ícone incorreto apresentado nas notificações"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:178
+msgid "Fix Desktop file being invalid"
+msgstr "Corrige o ficheiro do ambiente de trabalho de ser inválido"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:179
+msgid "Bump granite dependency version"
+msgstr "Impacto da versão de dependência do granite"
```

Comment:
"Bump" doesn't mean a literal collision in this context. It's jargon for a version upgrade.

Reply:
Yes I know. I don't usually translate these types of words but I always choose to leave the translations without anglicisms(as much Portuguese as possible). Do you have any suggestions for this string?

Reply:
I suppose "Atualização", or something to that effect, would work.